inspirehep/migrator/cli.py

```diff
@@ -77,7 +77,9 @@ def migrate_file(file_name, mirror_only=False, wait=False, force=False):
 
     populate_mirror_from_file(file_name)
     if not mirror_only:
-        migrate_from_mirror(wait_for_results=wait)
+        migration_tasks = migrate_from_mirror()
+        if wait:
+            migration_tasks.wait()
```

Comment:
Maybe add an
```
else:
    migration_tasks.forget()
```
otherwise results will accumulate in Redis.

Reply:
:+1:

Reply:
it's missing the `else:`